tests/update_eve_image/testdata/update_eve_image_oci.txt

```diff
@@ -0,0 +1,50 @@
+# Default EVE version to update
+{{$eve_ver := "6.0.0"}}
+
+# Obtain EVE version from environment variable EVE_VERSION
+{{$env := EdenGetEnv "EVE_VERSION"}}
+
+# If environment variable EVE_VERSION set, use it instead of default
+{{if $env}}{{$eve_ver = $env}}{{end}}
+
+# Obtain eve.hv from config
+{{$eve_hv := EdenConfig "eve.hv"}}
+
+# Obtain eve.arch from config
+{{$eve_arch := EdenConfig "eve.arch"}}
+
+# Combine variables into $short_version
+{{$short_version := printf "%s-%s-%s" $eve_ver $eve_hv $eve_arch}}
+
+# Use eden.lim.test for access Infos with timewait 30m
+{{$test := "test eden.lim.test -test.v -timewait 30m -test.run TestInfo"}}
+
+# Send command to update eveimage from OCI image
+message 'EVE update request'
+eden -t 10m controller edge-node eveimage-update oci://docker.io/lfedge/eve:{{$eve_ver}}-{{$eve_hv}}-{{$eve_arch}} -m adam://
+
+# Check stderr, it must be empty
+! stderr .
+
+# Run monitoring of Info messages to obtain info with PartitionState inprogress or active and previously defined ShortVersion
+message 'Waiting for EVE update...'
+{{$test}} -out InfoContent.dinfo.SwList[0].ShortVersion 'InfoContent.dinfo.SwList[0].PartitionState:inprogress|active InfoContent.dinfo.SwList[0].ShortVersion:{{ $short_version }}'
```

Comment:
Actually it is discussible, but, in case of subsequent revert, we should wait for active (not inprogress or active) state here. You can decrease testing time to do it faster like inside https://github.com/lf-edge/eden/blob/00efce00f4be016211969c8a69cdd403914b04f5/tests/update_eve_image/testdata/update_eve_image_http.txt#L23

WDYT, @deitch?

Reply:
I think I am ok either way. Whatever you want, either take this as is, or tell me you prefer with just `active` and I will change it

Reply:
I suggest you to change to active and modify testing time. Without it we just move our waiting onto nested script (revert_eve_image_update).

Reply:
Done.

Reply:
Thank you!